docs/gl_objects/merge_request_approvals.rst

```diff
@@ -58,7 +58,10 @@ Change project-level or MR-level MR approvals settings::
     p_mras.approvals_before_merge = 2
     p_mras.save()
 
-    mr_mras.set_approvers(approvals_required = 1)
+    mr.approvals.set_approvers(approvals_required = 1)
```

Comment:
Maybe we can fix the spacing while we're at it :)
```suggestion
    mr.approvals.set_approvers(approvals_required=1)
```

Reply:
I took the liberty of applying this so we dont block this for longer :sweat_smile: